Sources/NIOHTTP1/HTTPEncoder.swift

```diff
@@ -190,3 +182,377 @@ public final class HTTPResponseEncoder: ChannelOutboundHandler {
         }
     }
 }
+
+private extension ByteBuffer {
+    private mutating func write(status: HTTPResponseStatus) {
+        if case .ok = status {
+            // Optimize for 200 ok, which should be the most likely code (...hopefully).
+            self.write(staticString: status200)
```

Comment:
Do we actually need this? I don't think we'll ever enter this branch, because it requires 1.2 or higher to get to.

Reply:
I think this is safe to remove :)